Add blog post related to trademark policy draft (#1093)
rust-lang/blog.rust-lang.org: posts/inside-rust/2023-04-12-trademark-policy-draft-feedback.md

---
layout: post
title: "A note on the Trademark Policy Draft"
author: Ryan Levick, Jane Losare-Lusby, Tyler Mandry, Mark Rousskov, Josh Stone, and Josh Triplett
---

# A note on the Trademark Policy Draft

For the past eight months, the Rust Foundation Project Directors have been working with the informal Trademark Working Group and the Foundation staff to draft an updated policy and FAQ for the Rust trademark. We'd like to provide some context around this work and address the community response to the [feedback form] for the recently circulated draft.

## Background

Back when the Rust Foundation was created, one of the first things to happen was Mozilla transferring its ownership of the Rust trademark to the newly created foundation. An update to the existing policy[^mozilla-policy] was needed, and project leadership planned for this to be done together with the Foundation.

[^mozilla-policy]: https://prev.rust-lang.org/en-US/legal.html

> Since we wanted to incorporate community input to this policy and make it accessible as possible, the board waited until the Foundation was well-staffed to coordinate the effort. That included things like running a [community-wide survey] and discussing the result with a number of stakeholders, including the board, community members, project leadership, as well as legal counsel. The latest state of this process is the draft that was published late last week.

Reviewer: `[community-wide survey]` here seems to be missing its http link ?

Reviewer: Fixed in https://github.com/rust-lang/blog.rust-lang.org/pull/1094

---
layout: post

> title: "Seeking Feedback on a Trademark Policy Draft"

Author: ```suggestion
title: "A note on the Trademark Policy Draft"
```

author: Ryan Levick, Jane Losare-Lusby, Tyler Mandry, Mark Rousskov, Josh Stone, and Josh Triplett

> team: The Foundation Project Directors and informal Trademark Working Group Members

Reviewer 2: ```suggestion
```

CI is not happy with this

---

> # Seeking Feedback on a Trademark Policy Draft

Reviewer 2: ```suggestion
# A note on the Trademark Policy Draft
```

I meant to update this since we've already announced we're seeking feedback; this is more about the background and addressing the abuse directed at Foundation staff.

> The Rust Foundation Project Directors have been working with the informal Trademark Working Group and the Foundation staff to draft an updated policy and FAQ for the Rust trademark. We'd like to provide some context around this work and address the community response to the [feedback form] for the recently circulated draft.

Reviewer 3: Is it worth calling out that this process has been going on for several months? (And is not solely in reaction to posting of the draft)

Reviewer 3: I concerned that without an explicit call out, it could be interpreted as a reaction to the current controversy, rather than something that has been ongoing for quite some time.

Reviewer 2: Good point,

```suggestion
For the past eight months, the Rust Foundation Project Directors have been working with the informal Trademark Working Group and the Foundation staff to draft an updated policy and FAQ for the Rust trademark. We'd like to provide some context around this work and address the community response to the [feedback form] for the recently circulated draft.
```

## Next steps

We want to thank the community for participating in this process, and for your patience as we learn the best way to navigate it. We recognize that the process and communication around it could have been better. Notably, the wider project was insufficiently included in the process. We were  responsible for that and apologize.

We're committed to learning everything we can from this process and your feedback, and to talking as openly as we can about what we've learned. To that end, we will soon conduct and publish a retrospective around how the process unfolded.

Thank you again to those who have shared their thoughts on the Rust Trademark Policy draft respectfully. A summary of the feedback received will be shared after the consultation period closes. If you have not yet reviewed the draft, we invite you to fill out the [feedback form] by April 16 at 5 PM PDT. We only ask that you treat everyone in this community, including the Rust Foundation team, respectfully when doing so.

> Sincerely, (project directors and wg members add your names please)

Reviewer 4: ```suggestion
Sincerely,
```

> Sincerely, (project directors and wg members add your names please)

Reviewer 2: ```suggestion
Sincerely,
```
